projects/dotcms-js/src/lib/core/core-web.service.ts

```diff
@@ -135,77 +150,104 @@ export class CoreWebService {
      */
     public requestView(options: RequestOptionsArgs): Observable<ResponseView> {
         const request = this.getRequestOpts(options);
-
-        return this._http.request(request).pipe(
-            map((resp) => {
-                if (resp.json().errors && resp.json().errors.length > 0) {
+        return this.http.request(request).pipe(
+            filter(<T>(event: HttpEvent<T>) => event.type === HttpEventType.Response),
+            map((resp: HttpResponse<any>) => {
+                if (resp.body && resp.body.errors && resp.body.errors.length > 0) {
```

Comment:
```suggestion
                if (!!resp?.body?.errors?.length) {
```

Reply:
same as:
https://github.com/dotCMS/core-web/pull/1429/files#r493922046

Reply:
Very weird... maybe you version of tsc?

Reply:
https://github.com/dotCMS/core-web/pull/1429/files#r496092990